routes/web.php

```diff
@@ -1,16 +1,13 @@
 <?php
 
-/*
-|--------------------------------------------------------------------------
-| Web Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register web routes for your application. These
-| routes are loaded by the RouteServiceProvider within a group which
-| contains the "web" middleware group. Now create something great!
-|
-*/
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Route;
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get(
+    '/',
+    function () {
+        return view('welcome');
+    }
+);
+
+Auth::routes();
```

Comment:
I am confuse. When I load this branch I see there are login and register buttons on the homepage but they both take me to error pages 😢

Reply:
I know there is no frontend for this yet but is there an access point to login with yet?

Reply:
> I know there is no frontend for this yet but is there an access point to login with yet?

Yes,

If you `odm artisan route:list` you'll see the routes, they're just bootstrapped into this method.

I didn't delete the `welcome.blade.php` file because its in the project already, but I didn't add the login or register routes, because I know you haven't decided on a frontend scaffolding yet.

You should still be able to send a request to these access points though in the mean time through a client.